### PR TITLE
Bump src-cli to latest release

### PR DESCRIPTION
**Pull request by courier-new**

### CODE REVIEW

File: internal/src-cli/consts.go [hunk 1]
```diff
@@ -6,4 +6,4 @@ package srccli
 // as the suggested download with this instance.
 //
 // At the time of a Sourcegraph release, this is always the latest src-cli version.
-const MinimumVersion = "3.35.0"
+const MinimumVersion = "3.35.2"
```
Comment (LawnGnome) [inline internal/src-cli/consts.go:9]: Should this be 3?

```suggestion
const MinimumVersion = "3.35.3"
```

**Review by LawnGnome — APPROVED** [review]

**Review by eseliger — APPROVED** [review]

File: internal/src-cli/consts.go (hunk 1 above)
Comment (courier-new) [reply internal/src-cli/consts.go:9]: 🤦‍♀️ yes

Comment (eseliger) [reply internal/src-cli/consts.go:9]: 👁️

**Comment by eseliger** [comment]
Untitled

**Pull request merged**